sdk/appconfiguration/app-configuration/package.json

```diff
@@ -60,17 +60,25 @@
   },
   "sideEffects": false,
   "autoPublish": false,
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/generated/src/appConfigurationContext.ts",
+        "prefix": "packageVersion"
+      }
+    ]
+  },
   "dependencies": {
     "@azure/core-asynciterator-polyfill": "^1.0.0",
     "@azure/core-http": "^1.0.0",
     "@azure/core-paging": "^1.0.0",
-    "@azure/identity": "^1.0.0",
     "@azure/core-tracing": "1.0.0-preview.7",
+    "@azure/identity": "^1.0.0",
     "@opentelemetry/types": "^0.2.0",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
-    "@microsoft/api-extractor": "^7.5.4",
+    "@microsoft/api-extractor": "7.7.11",
```

Comment:
I had to stick api-extractor to 7.7.11, just how we're doing on master.

Reply:
I was able to revert this!

Reply:
Not doing this causes an identity build issue, even on the base branch hotfix/keyvault-challengeAuth-keys-secrets-4.0.3 . Mike and I are investigating.

Reply:
Seems like `rush install` cannot be trusted to work over time. This is a separate problem that Mike is investigating, unrelated to this hotfix.